src/cases/containers/case-hearings/case-hearings.component.ts

```diff
@@ -141,6 +142,7 @@ export class CaseHearingsComponent implements OnInit, OnDestroy {
             );
           }
           const caseHearingViewModels: HearingListViewModel[] = this.calculateEarliestHearingDate(caseHearingModels);
+          this.showLoadSpinner = false;
```

Comment:
This feels like quite a random place to do this. We talked about doing it at the end of the ngOnInit - was there a reason for it being here?